src/core/engines/engine_dispatcher.cpp

```diff
@@ -1,33 +1,32 @@
 #include "engine_dispatcher.hpp"
 
-namespace otto::engines {
-
-  // EngineDispatcher Implementations /////////////////////////////////////////
-  template<EngineType ET>
-  EngineDispatcher<ET>::EngineDispatcher()
-  {}
-  
-  template<EngineType ET>
-  void EngineDispatcher<ET>::init()
-  {
-    _engines = create_engines<ET>();
-    if (_engines.empty()) {
-      throw util::exception(
-        "No engines registered. Can't construct EngineDispatcher");
-    }
-  }
+#include "engine_selector_screen.hpp"
 
-  template<EngineType ET>
-  Engine<ET>& EngineDispatcher<ET>::current() noexcept
-  {
-    return *_current;
-  }
+namespace otto::engines {
 
-  template<EngineType ET>
-  const Engine<ET>& EngineDispatcher<ET>::current() const noexcept
-  {
-    return *_current;
-  }
+// EngineDispatcher Implementations /////////////////////////////////////////
+template<EngineType ET>
+void EngineDispatcher<ET>::init()
+{
```

Comment:
Is the indentation in this file as screwy as it looks in Github?

Reply:
woops yeah